provisioning/openstack-docker-client/README.md

```diff
@@ -104,3 +104,54 @@ systemctl restart docker
 ```
 
 Reboot the machine or log out/log in to reload your environment and complete the configurations.
+
+**Issue #3**
+
+This is likely a somewhat unique situation whereas the Docker Container is uanble to contact hosts while you are connected to a VPN.  The issue may manifest itself in different ways, but you should be able to validate whether or not you have an issue by executing a simple ping of your OpenStack API host.
```

Comment:
spelling correction..
---
the Docker Container is _unable_ to contact hosts
---